File: output/CodeGen/avatar/Python/Java_repair/codeforces_108_B.java
import java.util.*;

public class codeforces_108_B {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] arr = new int[n];
        for(int arr_i=0; arr_i < n; arr_i++){
            arr[arr_i] = in.nextInt();
        }
        in.close();
        Arrays.sort(arr);
        for(int i=1; i<n; i++){
            if(arr[i]!= arr[i-1] && arr[i] < arr[i-1]*2){
                System.out.println("YES");
                return;
            }
        }
        System.out.println("NO");
    }
}

File: output/ChatGPT/avatar/Python/C++/codeforces_108_B.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

string s() {
    int n;
    cin >> n;
    vector<int> a(n);
    for (int i = 0; i < n; i++) {
        cin >> a[i];
    }
    sort(a.begin(), a.end());
    for (int i = 1; i < n; i++) {
        if (a[i] != a[i - 1] && a[i] < a[i - 1] * 2) {
            return "YES";
        }
    }
    return "NO";
}

int main() {
    cout << s() << endl;
    return 0;
}